### PR TITLE
Feature - add manual link

### PR DESCRIPTION
**Pull request by joefitter**
Added manual link to first page, presist postcode if entered
Added default auth - process.env.POSTCODE_AUTH
Moved change postcode link to one line - added default text
Added template for postcode - has manual link

### CODE REVIEW

**Review by easternbloc — APPROVED** [review]

File: lib/index.js [hunk 1]
```diff
@@ -7,8 +7,17 @@ const _ = require('lodash');
 const DefaultModel = require('./default-model');
 const defaults = require('./defaults');
 
-const getFields = key => ({
+const conditionalTranslate = (key, t) => {
```
Comment (JoeChapman) [inline lib/index.js:10]: feels like there should be a default translate function

File: lib/index.js [hunk 2]
```diff
@@ -138,13 +143,15 @@ module.exports = config => {
 
       if (postcodeApiMessageKey) {
         const key = `pages.address-lookup.postcode-api.${postcodeApiMessageKey}`;
-        postcodeError = req.translate(key, req.translate, { default: null }) ||
+        postcodeError = conditionalTranslate(key, req.translate, req.translate) ||
```
Comment (JoeChapman) [inline lib/index.js]: does `conditionalTranslate` except a third param or is this a mistake?

File: lib/index.js (hunk 1 above)
Comment (joefitter) [reply lib/index.js:10]: yes there should be - i18nFuture will return the key if it fails, sometimes we want nothing, but I'm not fixing i18n, not today anyway

File: lib/index.js (hunk 2 above)
Comment (joefitter) [reply lib/index.js]: the latter - good spot!

Comment (joefitter) [reply lib/index.js]: @JoeChapman changed

**Pull request merged**